redash/query_runner/clickhouse.py

```diff
@@ -1,15 +1,21 @@
 import logging
 import re
 from urllib.parse import urlparse
+from uuid import uuid4
 
 import requests
 
 from redash.query_runner import *
+from redash.query_runner.databricks import split_sql_statements
```

Comment:
```suggestion
from redash.query_runner import split_sql_statements
```

Reply:
Fixed, rebased, and verified - it still works!

Reply:
Thank you!